projects/dotcms-field-elements/src/components/dot-select/dot-select.tsx

```diff
@@ -44,12 +45,28 @@ export class DotSelectComponent {
     _dotTouched = false;
     _dotPristine = true;
 
+    fieldAttr = {
+        type: 'dot-select',
+        name: this.name
+    };
+
     componentWillLoad() {
-        this._options = getDotOptionsFromFieldValue(this.options);
+        this.fieldAttr.name = this.name;
+        this.optionsWatch();
         this.emitInitialValue();
         this.emitStatusChange();
     }
 
+    @Watch('options')
+    optionsWatch(): void {
+        const validOptions = dotPropValidator({
+            field: { ...this.fieldAttr },
+            name: 'options',
+            value: this.options
+        }) || '';
```

Comment:
If we have the default we have to change it in 2 places, anyway we can have some sort of default value `const` or something?

Reply:
by creating a 
![image](https://user-images.githubusercontent.com/37185433/57795389-0d1b2200-7703-11e9-8ec9-1458aa48c73a.png)

and then setting that `CONST` in different places the doc ends up like this

![image](https://user-images.githubusercontent.com/37185433/57795377-068caa80-7703-11e9-9363-9017b8abffe9.png)

Reply:
maybe in a util, something like TyoeUtil.STRING_DEFAULT_VALUE
Also this value should be return by dotPropValidator

Reply:
not sure about `STRING_DEFAULT_VALUE` or maybe something like `LABEL_DEFAULT_VALUE`, `HINT_DEFAULT_VALUE`, etc...